api/src/services/server.ts

```diff
@@ -103,6 +103,10 @@ export class ServerService {
 			}
 		}
 
+		if (this.accountability?.user) {
+			info['showAdminOnboarding'] = toBoolean(env['SHOW_ADMIN_ONBOARDING']);
+		}
+
```

Comment:
Seems like we already have the same `if` for user above. Can we can move it in there?

Reply:
Oriented myself at the second check above, but yeah youre right, technically we do redundant checks both for websockets and onboarding! :smile: